lib/myfinance/entities/receivable_account_collection.rb

```diff
@@ -0,0 +1,14 @@
+module Myfinance
+  module Entities
+    #
+    # A wrapper to Myfinance entities collection
+    #
+    class ReceivableAccountCollection < Collection
+      def build_collection
+        response.parsed_body.each do |attributes|
+          collection.push(Myfinance::Entities::ReceivableAccount.new(attributes['receivable_account']))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.